server.go

```diff
@@ -113,6 +113,8 @@ type Server struct {
 	spanSinks   []sinks.SpanSink
 	metricSinks []sinks.MetricSink
 
+	derivedMetrics samplers.DerivedMetricsProcessor
```

Comment:
Is there a reason this needs to be set on the server at all? It looks like we only use it to configure the sink.

Reply:
No, I guess this is a vestige of previous iterations. I'll remove.